cli.go

```diff
@@ -4,10 +4,32 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"sort"
 )
 
+type allGroup struct {
+	Hosts []string               `json:"hosts"`
+	Vars  map[string]interface{} `json:"vars"`
+}
+
+func appendUniq(strs []string, item string) []string {
+	if len(strs) == 0 {
+		strs = append(strs, item)
+		return strs
+	}
+	sort.Strings(strs)
+	i := sort.SearchStrings(strs, item)
+	if i < len(strs) && strs[i] != item {
```

Comment:
According to this goDocs: https://golang.org/pkg/sort/#SearchStrings, this conditional excludes IP ranges that would go at the end of the list. Consider something like: 
	`if i == len(strs) || (i < len(strs) && strs[i] != item) {`